Description: Fix python interpreter discovery (Closes: #983140)
 On machines upgrade from buster to bullseye, the wrong interpreter will be
 discovered and used, causing the playbook run to fail. Update the fallback list
 so it correctly picks python3.9 on bullseye when both 3.9 and 3.7 are present,
 which usually is the case on freshly upgrade machines.
Origin: backport, https://github.com/ansible/ansible/commit/c986cbb9961bfaedf1a6ae7f0c2e34be26d9ab12
Forwarded: not-needed
Applied-Upstream: https://github.com/ansible/ansible/commit/c986cbb9961bfaedf1a6ae7f0c2e34be26d9ab12
Reviewed-by: Lee Garrett <debian@rocketjump.eu>
Last-Update: 2021-03-23
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/ansible/config/base.yml
+++ b/lib/ansible/config/base.yml
@@ -1523,6 +1523,8 @@
   name: Ordered list of Python interpreters to check for in discovery
   default:
   - /usr/bin/python
+  - python3.9
+  - python3.8
   - python3.7
   - python3.6
   - python3.5
